kolibri/content/utils/paths.py

```diff
@@ -16,6 +16,8 @@
 # set of file extensions that should be considered zip files and allow access to internal files
 POSSIBLE_ZIPPED_FILE_EXTENSIONS = set([".perseus", ".zip", ".epub", ".epub3"])
 
+def get_content_file_name(obj):
+    return '{checksum}.{extension}'.format(checksum=obj.id, extension=obj.extension)
```

Comment:
More general question, but this reminded me: It's an edge case, but in principle we could have files with the same checksum but different extensions. See any way we can avoid having that bite us with the `LocalFile` stuff, given the checksum is the pk?

In some ways, we should just have the filenames be the raw hash (no extensions), but that could mess with MIME types when we're serving the static content files, and it's also not back-compatible with what we have now.